sections/editing.include

```diff
@@ -1645,11 +1646,15 @@
 
   <div class="example">
     In this example, an author has suggested that a button should be available using a shortcut,
-    and suggested "C" as a memorable and useful shortcut.
+    and suggested "я" (the cyrillic letter "ya") as a memorable and useful shortcut in a document in russian.
```

Comment:
The language of the spec is English, so the Cyrillic characters need to be marked up with the correct lang attribute.